release notes/v0.8.1.md

```diff
@@ -0,0 +1,20 @@
+This is a patch release that fixes a few bugs, including one [regression bug](https://github.com/grafana/xk6-browser/issues/749) introduced in v0.8.0, adds one improvement on documentation, and does minor internal modifications.
+
+
+## Bugs fixed
+
+- Fix mapping of Page's Keyboard, Mouse, and Touchscreen properties. [#751](https://github.com/grafana/xk6-browser/pull/751)
```

Comment:
Almost there i think 🙂 

Atm, we put the issue in brackets after a period but not end it with a period:

e.g.

```suggestion
- Fix mapping of Page's Keyboard, Mouse, and Touchscreen properties. ([#751](https://github.com/grafana/xk6-browser/pull/751))
```

Reply:
Agree on the periods.
About the brackets, we did not follow that format in the last release, although we did in the previous ones. I prefer using brackets thought, so will change it.

Reply:
Fixed in: [b5994e3](https://github.com/grafana/xk6-browser/pull/775/commits/b5994e37efaa6064d328a025a4673deae8c4b159).

Reply:
> About the brackets, we did not follow that format in the last release

🤦 I missed that 😅